packages/react-sdk/README.md

```diff
@@ -28,6 +28,7 @@ These ready-made components can help you quickly build a chat app.
 
 - Node 16.10+
 - React 16.14+
+- For use within this repo, Yarn v3+. See [Yarn Installation](https://yarnpkg.com/getting-started/install).
```

Comment:
sorry again for this confusion on this. Yarn V3 is only required at the repo root level and when developing examples and packages in this repo. all external consumption of the React SDK does not have this requirement.